ledger/participant-integration-api/src/main/scala/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -57,7 +58,8 @@ object StandaloneApiServer {
       engine: Engine,
       servicesExecutionContext: ExecutionContextExecutor,
       checkOverloaded: TelemetryContext => Option[state.SubmissionResult] =
-        _ => None, // Used for Canton rate-limiting
+        _ => None, // Used for Canton rate-limiting,
+      commandDeduplicationFeatures: CommandDeduplicationFeatures,
```

Comment:
Could this be exposed by the `WriteService` instead?

Reply:
I had the same question in mind as well.
Even though it feels cleaner in theory I decided not to go that way because of the following reasons:
- Not all the "features" we plan on adding are dependent on the write service. For example, the feature which lets us know if we have participant side deduplication cannot be determined in the write service
- It feels like we are polluting the write service by adding a "read" layer to it